devlib/target.py

```diff
@@ -1652,6 +1652,9 @@ def __init__(self,
                                                shell_prompt=shell_prompt,
                                                conn_cls=conn_cls)
 
+    def wait_boot_complete(self, timeout=10):
+        pass
```

Comment:
Should this be added to `LinuxTarget`?

Reply:
Yup, added.

Reply:
Does this need to be in LocalLinuxTarget as well?

Reply:
Good point. Removed from `LocalLinuxTarget`